react/theme.css

```diff
@@ -0,0 +1,54 @@
+body {
+    width: 100%;
+}
+.top-100 {
+    top: 100%
+}
+
+.top-50 {
+    top: 50%
+}
+
+.minicart-item {
+    width: 350px;
+    height: 90px;
+}
+
+.image-size {
+    width: 70px;
+    height: 70px;
+}
+
+.name-size {
+    width: 264px;
+    height: 50px;
+}
+
+.sku-size {
+    width: 138px;
+    height: 10px;
+}
+
+.minicart {
+    display: block;
+    visibility: visible;
+}
+
+@media only screen and (max-width: 600px) {
+    .minicart {
+        display: none;
+        visibility: hidden;
+    }
+}
+
+.minicart-content {
+    height: 300px;
+}
+
+.arrow-up {
+    width: 0; 
+    height: 0; 
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid white;
+  }
```

Comment:
Put the component css separate (minicart.css, for example) and import it in the component file